src/Elmish.WPF/ViewModel.fs

```diff
@@ -152,6 +152,16 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
       log "[%s] ErrorsChanged \"%s\"" propNameChain propName
       errorsChanged.Trigger([| box this; box <| DataErrorsChangedEventArgs propName |])
 
+  let updateValidationError name model = function
+    | TwoWayValidate { Validate = validate } ->
+        match validate model with
+        | ValueNone -> removeError name
+        | ValueSome error -> setError error name
+    | _ -> ()
+
+  let setInitialValidationError name binding = updateValidationError name initialModel binding
+  let updateValidationError name binding = updateValidationError name currentModel binding
```

Comment:
Are these necessary? Since they're only used once each, I think it's more clear if they are removed.

Reply:
Nope.  I removed them and swapped the order of the `model` and `name` arguments.  I merged this change with the previous commit and force pushed.